changelog.mdx

```diff
@@ -4,6 +4,38 @@ description: "All the nice stuff you asked for"
 icon: "sparkles"
 ---
 
+## 2024-01-03
+
+- You can add new steps to draft test cases now
+- `Bounding boxes`: Element that was interacted with is highlighted by a bounding box in test step screenshots
+
+<Frame caption="Element that was interacted with is highlighted by a bounding box, screenshot 01/2024">
+  <img
+    src="images/changelog/bounding-box.png"
+    alt="bounding boxes in test steps screenshot"
+  />
+</Frame>
+
+## 2023-12-18
+
+- Hanging tests runs mitigated
```

Comment:
```suggestion
- Fixed: Sometimes some of our test runs got stuck, we improved this and no stuck tests should occurr anymore.
```

Reply:
I think i misspelled occur I think only one r, sorry 😬

Reply:
that's what happens when you listen to Cardi B for too long